autosklearn/util/parallel.py

```diff
@@ -0,0 +1,13 @@
+import multiprocessing
+import sys
+
+
+def preload_modules(context: multiprocessing.context.BaseContext) -> None:
+    all_loaded_modules = sys.modules.keys()
+    preload = [
+        loaded_module for loaded_module in all_loaded_modules
+        if loaded_module.split('.')[0] in (
+            'smac', 'autosklearn', 'numpy', 'scipy', 'pandas', 'pynisher', 'sklearn',
```

Comment:
I am just wondering why not configspace also?

Reply:
But I think I got my answer because it only loads the models up to this point :)

Reply:
That was oversight, thanks for spotting. Not having to load the ConfigSpace obviously is a huge plus.